server/api/rounds.py

```diff
@@ -295,11 +295,12 @@ def calculate_risk_measurements(election: Election, round: Round):
         contest = round_contest.contest
 
         if election.audit_type == AuditType.BALLOT_POLLING:
+            assert election.ballot_polling_type is not None
```

Comment:
why do you need this assertion here but not in `sample_sizes.py` below?

Reply:
The type checker was having issues with it (without this it sees it as `Optional` but it expects not `Optional`

Reply:
yeah that's what i figured. i'm just surprised it didn't have the same issue in sample_sizes